### DISCUSSION TITLE
CAF in the Post Production Industry

### DISCUSSION THREAD

**Discussion by alatdneg** [actor-framework/actor-framework#1349] [Show and tell]
Been working on this project for over two years..  :)

https://www.dneg.com/xstudio/

**Comment by Neverlord** [comment]
Thanks for sharing, this looks really impressive! 👏

**Comment by alatdneg** [comment]
My thanks to you :) 

CAF made life so much easier, I can only imagine the pain we would have felt if we went the Mutex route.
